apps/web/src/server/applications/create/applicant/applications-create-applicant.types.d.ts

```diff
@@ -34,6 +34,26 @@ export type ApplicationsCreateApplicantFullNameBody = {
 	applicantlastName?: string;
 };
 
+export type ApplicationCreateApplicantAddressStage =
+	| 'searchPostcode'
+	| 'selectAddress'
+	| 'manualAddress';
+export type ApplicationsCreateApplicantAddressProps = {
+	formStage: ApplicationCreateApplicantAddressStage;
+	errors?: Record<string, { msg: string }> | ValidationErrors;
+	postcode: string;
+	addressList?: ApplicationAddress[];
+};
+
+export type ApplicationsCreateApplicantAddressBody = {
```

Comment:
i'm not proud of this, but this page is actually three pages so the structure of `body` can be quite diffrent